package javaOOP.Exception;

public class TryCatchException {

	public static void main(String[] args) {
		int number = 10;

	}

}
